src/pages/Cabinet/components/CabinetModal.tsx

```diff
@@ -80,6 +96,16 @@ export const CabinetModal: FC<CabinetModalProps> = ({ open, onUpdate, onClose })
     },
   ];
 
+  useEffect(() => {
+    if (error) {
```

Comment:
check  error  i n another  one  useEffect
one  hook for  error  one  for  success